### DISCUSSION TITLE
Display issue of PointCloudLayer in OrbitView

### DISCUSSION THREAD

**Discussion by ethanruiyu** [visgl/deck.gl#6712] [Q&A] [answered]
When the point cloud rotation center is at its structural center, the point cloud map can be rotated properly, and we can zoom in anywhere we wish to; However, if we drag the point cloud to the side, and then rotate it to make it move further from the screem direction, the cloud point will look much thinner, and the points will completely disappear as we zoom in closer from this angle.
![iShot2022-02-28 16 30 00](https://user-images.githubusercontent.com/81229588/155952792-45a1baa1-f9c4-4a11-b34c-5d3ff18a1c5b.gif)

**Comment by Pessimistress** [comment, marked as answer]
See https://github.com/visgl/deck.gl/discussions/6696#discussioncomment-2269035
